pkg/agent/cniserver/server.go

```diff
@@ -710,6 +712,16 @@ func (s *CNIServer) interceptAdd(cniConfig *CNIConfig) (*cnipb.CniCmdResponse, e
 		return &cnipb.CniCmdResponse{CniResult: []byte("")}, fmt.Errorf("failed to connect container %s to ovs: %w", cniConfig.ContainerId, err)
 	}
 
+	if s.multiClusterEnabled {
```

Comment:
NoEncap mode will not call this func. I think you should change the MTU calculation path for noEncap mode.

Reply:
Sure, I will refine this part and address remaining comments when I finished verification and refactor. thanks.

Reply:
For noEncap mode, I refined the method `getNodeMTU` to deduct MTU when enableMulticluster is true.